### PR TITLE
Prepare footer

### PR DESCRIPTION
**Pull request by 38programmer61**
Footer element which contains contact information as an example has been added to the system.

### CODE REVIEW

File: front-end/playground/src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ 
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <SidebarComponent />
+      <h1>
+        Main Content 
+        <br /> <br /> <br /> <br /><br /> <br /> <br /> <br /><br /> <br /> <br /> <br /><br /> <br /> <br /> <br />
```
Comment (eye-78) [inline front-end/playground/src/App.js:16]: We can use flex layout. [See here](https://reactnative.dev/docs/flexbox)

File: front-end/playground/src/PermanentComponents/Footer.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+function Footer() {
+    return(
+        <div>
+
+
+            <Card bg="danger" className="footer">
+                <Card.Header  >Contact Information</Card.Header>
+                <Card.Body>
+                    <blockquote className="blockquote mb-0">
+                    <p>
```
Comment (eye-78) [inline front-end/playground/src/PermanentComponents/Footer.js:14]: Flexbox provides "space-around" functionality

**Review by eye-78 — APPROVED** [review]
Nice start. Thanks for your efforts. Please look at my comments

**Review by salihozcelik — APPROVED** [review]

File: front-end/playground/src/PermanentComponents/Footer.js (hunk 2 above)
Comment (38programmer61) [reply front-end/playground/src/PermanentComponents/Footer.js:14]: Thanks I will try to take necessary actions.

**Pull request merged**